lib/tests/streamlit/elements/map_test.py

```diff
@@ -73,6 +71,16 @@ def test_alternative_names_columns(self, lat_column_name, lon_column_name):
         c = json.loads(self.get_delta_from_queue().new_element.deck_gl_json_chart.json)
         self.assertEqual(len(c.get("layers")[0].get("data")), 4)
 
+    def test_map_uses_convert_anything_to_df(self):
+        """Test that st.map uses convert_anything_to_df to convert input data."""
+        with mock.patch(
+            "streamlit.type_util.convert_anything_to_df"
+        ) as convert_anything_to_df:
+            convert_anything_to_df.return_value = df1
```

Comment:
nit: `df1` is not a good name for a global variable 🙂

Reply:
renamed